template/src/index.css

```diff
@@ -0,0 +1,21 @@
+:root {
+  font-family: "Poppins", sans-serif;
+  font-size: calc(10px + 2vmin);
```

Comment:
what does `vmin` do?

Reply:
vertical min height I believe? This is the same css as the create-react-project, so this is admittedly a copy-paste